beets/ui/commands.py

```diff
@@ -873,102 +871,66 @@ def import_files(lib, paths, query):
     plugins.send('import', lib=lib, paths=paths)
 
 
-def import_func(lib, opts, args):
+# TODO: add 'imp' and 'im' aliases
+@click.command('import', short_help='import new music')
+@click.option('copy', '-c', '--copy', flag_value=True,
+              help='copy tracks into library directory (default)')
+@click.option('copy', '-C', '--nocopy', flag_value=False, default=None,
+              help="don't copy tracks (opposite of -c)")
+@click.option('write', '-w', '--write', flag_value=True,
+              help="write new metadata to files' tags (default)")
+@click.option('write', '-W', '--nowrite', flag_value=False, default=None,
+              help="don't write metadata (opposite of -w)")
+@click.option('autotag', '-a', '--autotag', flag_value=True,
+              help='infer tags for imported files (default)')
+@click.option('autotag', '-A', '--noautotag', flag_value=False, default=None,
+              help="don't infer tags for imported files (opposite of -a)")
+@click.option('resume', '-p', '--resume', flag_value=True,
+              help='resume importing if interrupted')
+@click.option('resume', '-P', '--noresume', flag_value=False, default=None,
+              help='resume importing if interrupted')
+@click.option('-q', '--quiet', is_flag=True, default=None,
+              help='never prompt for input: skip albums instead')
+@click.option('-l', '--log', metavar='LOG', default=None,
+              help='file to log untaggable albums for later review')
+@click.option('-s', '--singletons', is_flag=True, default=None,
+              help='import individual tracks instead of full albums')
+@click.option('-t', '--timid', is_flag=True, default=None,
+              help='always confirm all actions')
+@click.option('-L', '--library', metavar='LIBRARY', default=None,
+              help='retag items matching a query')
+@click.option('incremental', '-i', '--incremental', flag_value=True,
+              help='skip already-import directories')
+@click.option('incremental', '-I', '--noincremental', flag_value=False,
+              default=None, help='do not skip already-imported directories')
```

Comment:
I think this can be replaced with http://click.pocoo.org/4/options/#boolean-flags